lib/DDGC/DB/Result/User/Blog.pm

```diff
@@ -1,202 +1,108 @@
-	data_type => 'bigint',
-	is_nullable => 0,
+    data_type   => 'bigint',
+    is_nullable => 0,
 };
 
 column translation_of_id => {
-	data_type => 'bigint',
-	is_nullable => 1,
+    data_type   => 'bigint',
+    is_nullable => 1,
 };
 
 column title => {
-	data_type => 'text',
-	is_nullable => 0,
+    data_type   => 'text',
+    is_nullable => 0,
 };
 
 column uri => {
-	data_type => 'text',
-	is_nullable => 0,
+    data_type   => 'text',
+    is_nullable => 0,
 };
 
 column teaser => {
-	data_type => 'text',
-	is_nullable => 0,
+    data_type   => 'text',
+    is_nullable => 0,
 };
-sub html_teaser {
-	my ( $self ) = @_;
-	return $self->teaser if $self->raw_html;
-	return $self->ddgc->markup->html($self->teaser);
-}
 
 column content => {
-	data_type => 'text',
-	is_nullable => 0,
+    data_type   => 'text',
+    is_nullable => 0,
 };
-sub html {
-	my ( $self ) = @_;
-	return $self->content if $self->raw_html;
-	return $self->ddgc->markup->html($self->content);
-}
 
 column topics => {
-	data_type => 'text',
-	is_nullable => 1,
-	serializer_class => 'JSON',
+    data_type        => 'text',
+    is_nullable      => 1,
+    serializer_class => 'JSON',
 };
 
 column company_blog => {
-	data_type => 'int', # bool
-	is_nullable => 0,
-	default_value => 0,
+    data_type     => 'int',
+    is_nullable   => 0,
+    default_value => 0,
 };
 
 column raw_html => {
-	data_type => 'int', # bool
-	is_nullable => 0,
-	default_value => 0,
+    data_type     => 'int',
+    is_nullable   => 0,
+    default_value => 0,
 };
 
 column live => {
-	data_type => 'int', # bool
-	is_nullable => 0,
-	default_value => 0,
-	keep_storage_value => 1,
+    data_type          => 'int',
+    is_nullable        => 0,
+    default_value      => 0,
+    keep_storage_value => 1,
 };
 
 column seen_live => {
-	data_type => 'int', # bool
-	is_nullable => 0,
-	default_value => 0,
+    data_type     => 'int',
+    is_nullable   => 0,
+    default_value => 0,
 };
 
 column language_id => {
-	data_type => 'bigint',
-	is_nullable => 1,
+    data_type   => 'bigint',
+    is_nullable => 1,
 };
 
 column data => {
-	data_type => 'text',
-	is_nullable => 1,
-	serializer_class => 'JSON',
+    data_type        => 'text',
+    is_nullable      => 1,
+    serializer_class => 'JSON',
 };
 
 column fixed_date => {
-	data_type => 'timestamp with time zone',
-	is_nullable => 1,
+    data_type   => 'timestamp with time zone',
+    is_nullable => 1,
 };
 
 column created => {
-	data_type => 'timestamp with time zone',
-	set_on_create => 1,
+    data_type     => 'timestamp with time zone',
+    set_on_create => 1,
 };
 
 column updated => {
-	data_type => 'timestamp with time zone',
-	set_on_create => 1,
-	set_on_update => 1,
+    data_type     => 'timestamp with time zone',
+    set_on_create => 1,
+    set_on_update => 1,
 };
 
 belongs_to 'user', 'DDGC::DB::Result::User', 'users_id';
-belongs_to 'language', 'DDGC::DB::Result::Language', 'language_id', { join_type => 'left' };
```

Comment:
Can you briefly expound on what was going on here? It looks like transaction / triggers on column change. Where is the functionality now?